Tests/Integration/IntegrationTestFixtures+UrlLoadBalancing.swift

```diff
@@ -0,0 +1,267 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+@testable import MobileCoin
+import XCTest
+
+extension IntegrationTestFixtures {
+    static func createValidConsensusUrlLoadBalancer()
+    throws -> SequentialUrlLoadBalancer<ConsensusUrl> {
+        // same url purposefully doubled-up, as we use the current index to detect rotation
+        let urlStrings = [
+            IntegrationTestFixtures.network.consensusUrl,
+            IntegrationTestFixtures.network.consensusUrl,
+        ]
+        let consensusUrls = try ConsensusUrl.make(strings: urlStrings).get()
+        return SequentialUrlLoadBalancer(urls: consensusUrls)
+    }
+
+    static func createValidFogUrlLoadBalancer() throws -> SequentialUrlLoadBalancer<FogUrl> {
+        // same url purposefully doubled-up, as we use the current index to detect rotation
+        let urlStrings = [
+            IntegrationTestFixtures.network.fogUrl,
+            IntegrationTestFixtures.network.fogUrl,
+        ]
+        let fogUrls = try FogUrl.make(strings: urlStrings).get()
+        return SequentialUrlLoadBalancer(urls: fogUrls)
+    }
+
+    static func createConsensusUrlLoadBalancerWithInitialInvalidUrl()
+    throws -> SequentialUrlLoadBalancer<ConsensusUrl> {
+        let urlStrings = [
+            IntegrationTestFixtures.invalidConsensusUrl,
+            IntegrationTestFixtures.network.consensusUrl,
+        ]
+        let consensusUrls = try ConsensusUrl.make(strings: urlStrings).get()
+        return SequentialUrlLoadBalancer(urls: consensusUrls)
+    }
+
+    static func createFogUrlLoadBalancerWithInitialInvalidUrl()
+    throws -> SequentialUrlLoadBalancer<FogUrl> {
+        let urlStrings = [
+            IntegrationTestFixtures.invalidFogUrl,
+            IntegrationTestFixtures.network.fogUrl,
+        ]
+        let fogUrls = try FogUrl.make(strings: urlStrings).get()
+        return SequentialUrlLoadBalancer(urls: fogUrls)
+    }
+
+    static func createFogViewConnection
```

Comment:
Ya, the code, as is, where the open parens is on a newline, was due to the linter warning `Multline Parameters Brackets Violation: Multiline parameters should have their surrounding brackets in a new line. (multiline_parameters_brackets)`.  

It's a little bit misleading, because it'll give you that warning for both the open parens and the close parens if you, say, were to have the first param as a continuation of the function name line...  

So when going through this, while I thought it looked strange, I was trying to follow the `multiline_parameters_brackets` instructions, which literally stated the surrounding brackets should be in a new line for both open parens and close.

Wrapping the first param does get rid of the linter warning for open parens, so it's strange / buggy that it's flagged :/

Reply:
Fixed - seemed like the only issue was the open parens for multi-line parameter lists, where I blindly followed the linter...